### PR TITLE
RockX profile

### PR DESCRIPTION
**Pull request by tony-rockx**

### CODE REVIEW

File: profiles/RockX/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+Name: RockX
+Email: support@rockx.com
+Linkedin: https://www.linkedin.com/company/23771114/
+Twitter: https://twitter.com/rockx_official
+Medium: https://medium.com/rockx
+---
+
+NOTE: This is just a suggested template, modify it as you see fit.
```
Comment (mswilkison) [inline profiles/RockX/README.md:9]: Delete this line

File: profiles/RockX/profile.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "RockX",
+  "contact": {
+    "email": "support@rockx.com",
+    "linkedin": "https://www.linkedin.com/company/23771114/",
+    "twitter": "https://twitter.com/rockx_official",
+    "medium": "https://medium.com/rockx"
+  }
```
Comment (mswilkison) [inline profiles/RockX/profile.json:8]:
```suggestion
  },
```

**Comment by mswilkison** [comment]
Also there are merge conflicts since you removed the example directory. Can you put it back and we will remove in a separate commit. Thanks

**Comment by tony-rockx** [comment]
Thanks MacLane. Was just about to get to it but saw that you merged it.
Appreciate it

**Pull request merged**